R/bambu-quantify.R

```diff
@@ -40,6 +40,7 @@ bambu.quantify <- function(readClass, annotations, emParameters,ncore = 1,
             ncol = 1, dimnames = list(NULL, colNameRC)),
         theta = matrix(counts$theta, 
             ncol = 1, dimnames = list(NULL, colNameRC))), colData = colDataRC)
+    metadata(seOutput)$distTable = metadata(readClass)$distTable
```

Comment:
this will be copied to the final output but it might not be useful as they can't be linked to the original samples, also when working with large files this will increase the memory quite a bit. I would suggest to include an option in bambu something like returnDistTable which has to be set to TRUE for those to be returned, otherwise remove this before combining multiple SEs

Reply:
Now there is the returnDistTable argument part of bambu() which is false by default.
When true the final se object will have metadata(se)$distTables, which is a list of each samples distTables
resolved in e7831dbf3b1df3819b90b73657cbc5a083ee41c8